src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-elasticsearch-monitoring-integration.mdx

```diff
@@ -1,29 +1,29 @@
 tags:
   - Integrations
   - Amazon integrations
   - AWS integrations list
-metaDescription: 'New Relic''s Amazon Elasticsearch monitoring integration: what data it reports, and how to enable it.'
+metaDescription: "The New Relic Amazon OpenSearch Service monitoring integration: what data it reports, and how to enable it."
 redirects:
   - /docs/integrations/amazon-integrations/aws-integrations-list/aws-elasticsearch-monitoring-integration
   - /docs/aws-elasticsearch-integration
   - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-elasticsearch-integration
   - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-elasticsearch-monitoring-integration
 ---
 
-<Callout variant="important">
-  Enable the [AWS CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-metric-stream/) to monitor all CloudWatch metrics from your AWS services, including custom namespaces. Individual integrations are no longer our recommended option.
-</Callout>
-
-[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for reporting Amazon Elasticsearch data to New Relic. This document explains the integration's features, how to activate it, and what data can be reported.
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an integration for reporting Amazon OpenSearch Service data to New Relic. This document explains the integration's features, how to activate it, and what data can be reported.
 
 ## Features [#features]
 
-[Amazon Elasticsearch Service](https://aws.amazon.com/elasticsearch-service/) is a fully managed service that delivers Elasticsearch’s easy-to-use APIs and real-time capabilities along with the availability, scalability, and security required by production workloads. New Relic's Elasticsearch monitoring integration allows you to track cluster status, CPU utilization, read/write latency, throughput, and other metrics, at specific points in time. Elasticsearch data is also available [to query, analyze, and chart your data](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/).
+[Amazon OpenSearch Service](https://aws.amazon.com/elasticsearch-service/) (formerly Amazon Elasticsearch) is a fully managed service that delivers OpenSearch's easy-to-use APIs and real-time capabilities along with the availability, scalability, and security required by production workloads. Our OpenSearch monitoring integration allows you to track cluster status, CPU utilization, read/write latency, throughput, and other metrics, at specific points in time. This data is also available [to query, analyze, and chart your data](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/).
```

Comment:
url is the old one. AWS does the redirect to the new one but i would put the correct one: https://aws.amazon.com/opensearch-service/